test/spec/data/facet_spec.js

```diff
@@ -258,5 +258,24 @@ define(['test/mock', 'lodash'], function(mock, _) {
         });
       });
     });
+    describe('on uiClearFacets', function() {
+      beforeEach(function() {
+        this.component.trigger('config', mock.config);
+        this.component.trigger('data', mock.data);
+      });
+      it('emits a "dataFacets" event with no facets selected', function() {
+        this.component.trigger('uiClearFacets', {"facet" : "commudddnity"});
```

Comment:
Since there's a typo here, I suspect that this test would pass even without the `uiClearFacets` call.  What you want is to trigger an event to actually select some facets, and then the clear event.